libraries/cyclestreets-core/src/main/res/values-pt/arrays.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string-array name="units">
+    <item>Kilometres</item>
+    <item>Miles</item>
+  </string-array>
+  <string-array name="unit_values">
+    <item>km</item>
+    <item>miles</item>
+  </string-array>
+  <string-array name="speed">
+    <item>Unhurried 10mph (16km/h)</item>
+    <item>Cruising 12mph (20km/h)</item>
+    <item>Quick 15mph (24km/h)</item>
+  </string-array>
+  <string-array name="speed_values">
+    <item>16</item>
+    <item>20</item>
+    <item>24</item>
+  </string-array>
+  <string-array name="routetype">
+    <item>Quietest</item>
+    <item>Balanced</item>
+    <item>Fastest</item>
+    <item>Shortest</item>
+  </string-array>
+  <string-array name="routetype_values">
+    <item>quietest</item>
+    <item>balanced</item>
+    <item>fastest</item>
+    <item>shortest</item>
+  </string-array>
```

Comment:
Ok @oliverlockwood , another question : in this file, we have ```<string-array name="routetype">```  and ```<string-array name="routetype_values">``` with the same information, but one capitalized and the other not. It looks like only the first one should be translated, and the other will be used internally, is that right?